### ISSUE TITLE
[zabbix-community/zabbix] Integrate admin login credentials during the deployment (via helm chart)

### ISSUE THREAD

**Issue by nikable** [zabbix-community/helm-zabbix#49]
**Is your feature request related to a problem? Please describe.**
No, it is only a possible feature request. 

**Describe the solution you'd like**
Instead of manually entering the default admin credentials (provided in the Zabbix GitHub chart repository page), it would be simply great to have the option to customize Admin user login credentials within the Helm chart itself. This way, it can be bootstrapped during the deployment and would probably make the deployment and login environment more secure.

**Describe alternatives you've considered**
As an alternative, currently, we are manually entering the default Admin credentials for the first time and changing it later.

**Additional context**
This feature can be integrated within the Zabbix Web microservice.

**Comment by aeciopires** [comment]
Hello @nikable!

Thanks for reporting your issue, but unfortunately I don't know how to implement this. I would appreciate it if you could open a pull request with the solution

**Comment by aeciopires** [comment]
@nikable I found this page with the solution.

- https://bobcares.com/blog/zabbix-reset-admin-password/
- https://www.zabbix.com/documentation/current/en/manual/web_interface/password_reset

I tried to implement it using a script and mounting it on the cluster as configmap and created an initcontainer in the Zabbix Web pod... but changing the password didn't work... in fact I ended up corrupting the users table in the test environment and if it had worked, it would have been It's pretty bad to have to change the password every time the Zabbix Web pod is recreated (in Kubernetes things are ephemeral and this can happen at any time for N reasons).

I'll leave it as a challenge for anyone who wants to and knows how to implement it in an elegant way, like how it's done in Grafana's helm chart. https://github.com/grafana/helm-charts/tree/main/charts/grafana
